packages/design-system-dashboard-cli/src/find-ds-components.js

```diff
@@ -171,6 +215,7 @@ function findComponents(searchStrings) {
   let vwWebComponents;
   let contentBuildWC;
   let vwComponents;
+  const uswdsV3Components = [...search(vwModules, wcUswds3Regex)];
```

Comment:
I couldn't think of a good way to search for vw components and keep track of the uswds prop "in place", so just decided to search explicitly for them separately.